recipes/kedro-viz/package.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
This would force us to keep this `package.json` and the one upstream in sync, can't we get it from the repository?